frontend/test/metabase/scenarios/admin/datamodel/metadata.cy.spec.js

```diff
@@ -138,7 +138,15 @@ describe("scenarios > admin > datamodel > metadata", () => {
       .findByText("Search box")
       .click();
 
-    cy.reload();
+    openOptionsForSection("Display values");
+    popover()
+      .findByText("Custom mapping")
+      .should("not.exist");
+
+    openOptionsForSection("Filtering on this field");
+    popover()
+      .findByText("A list of all values")
+      .click();
```

Comment:
Add another assertion to ensure after the `A list of all values` options is selected, `Custom mapping` will appear again.